modules/rostests/apitests/atl/atlconv.cpp

```diff
@@ -0,0 +1,98 @@
+/*
+ * PROJECT:     ReactOS api tests
+ * LICENSE:     GPL-2.0+ (https://spdx.org/licenses/GPL-2.0+)
+ * PURPOSE:     Test for <atlconv.h>
+ * COPYRIGHT:   Copyright 2023 Katayama Hirofumi MZ <katayama.hirofumi.mz@gmail.com>
+ */
+
+#ifdef HAVE_APITEST
+    #include <apitest.h>
+#else
+    #include "atltest.h"
+#endif
+#include <atlbase.h>
+#include <atlconv.h>
+
+START_TEST(atlconv)
```

Comment:
If you use the `EX` version, with a template size of something like `5` or at least shorter than your default, does it work as well?

Reply:
Added.